src/modules/secrets/dtos/create-Secrets.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiProperty } from "@nestjs/swagger";
```

Comment:
@danielaKokoneshi309 Suggestion for file naming, use either:
- camelCase
- snake-case
- underscore
Only one of them everywhere, not mixed